docs/developer/products.mdx

```diff
@@ -337,6 +337,64 @@ type Money {
 }
 ```
 
+See [Prices](api-conventions/prices.mdx) for more information about money types.
```

Comment:
See [Prices](api-conventions/prices.mdx) for more information about the money types.